rskj-core/src/main/java/co/rsk/core/bc/BlockchainBranchComparator.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2019 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.core.bc;
+
+import org.ethereum.core.Block;
+import org.ethereum.db.BlockStore;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class BlockchainBranchComparator {
```

Comment:
What about `BlockchainForkCalculator`?

Reply:
I think it's `BlockFork` that's wrong. Maybe it could be... `BranchComparison`?

Reply:
My point was b/c of the existing concept in the Java platform for a comparator in the terms of `java.util.Comparator`. It's not crucial at all